contribute/documentation.md

```diff
@@ -7,7 +7,6 @@ title: Documentation Contributions
 There are several ways you can help out with the improvement of Scala documentation. These include:
 
 * API Documentation in Scaladoc
-* Code examples and tutorials in activator templates.
```

Comment:
Anyway, this is the wrong place to suggest these changes. I think we're making suggestions for improving the getting started page, which is not the file we're commenting on (we're commenting on a contributing page)